launchcontainer/static/js/src/launchcontainer.js

```diff
@@ -146,6 +152,8 @@ function LaunchContainerXBlock(runtime, element) {
                            + "<p> Please contact the administrator";
           if (supportEmail) {
             $final_msg += ' at <a href="mailto:' + supportEmail + '">' + supportEmail + '.</a>';
+          } else {
+            $final_msg += '<a href="/help">Contact us for support.</a>';
```

Comment:
A little bit of rewording and open in a new page:

```suggestion
            $final_msg += 'at <a href="/help" rel="noreferrer" target="_blank">help section</a>';
```